src/Teams/Data/Migrations/20200908210058_Updated.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Why is this migration is empty? It should contain description of the migration on Up and Down